docker-entrypoint.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -euo pipefail
+
+if [ $ENVIRONMENT = "local" ]; then
+    echo "$SERVICE_ACCOUNT_BASE64" | base64 -d >config/sa.json
```

Comment:
if we are doing local dev, inject the service account via an encoded var. and set the `GOOGLE_APPLICATION_CREDENTIALS `, this enables us to use kms decrypt (sops secrets) and to interact with the firebase datastore